<?xml version="1.0" encoding="UTF-8"?>
<robot generator="Robot 3.1.1 (Python 3.7.2 on win32)" generated="20190625 20:27:31.772" rpa="false">
<suite id="s1" name="TC 002 Pharmacy" source="C:\Users\TestingWorld\PycharmProjects\AngularWebAutomation\TestCases\TC_002_Pharmacy.robot">
<test id="s1-t1" name="Adding prescription in VIDA Plus application TC_001">
<kw name="User Open Browser" library="CommonKewwords">
<arguments>
<arg>${URL}</arg>
<arg>Chrome</arg>
</arguments>
<kw name="open browser">
<arguments>
<arg>${ApplicationUrl}</arg>
<arg>${BrowserName}</arg>
</arguments>
<msg timestamp="20190625 20:27:32.299" level="FAIL">No keyword with name 'open browser' found.</msg>
<status status="FAIL" starttime="20190625 20:27:32.298" endtime="20190625 20:27:32.299"></status>
</kw>
<status status="FAIL" starttime="20190625 20:27:32.280" endtime="20190625 20:27:32.300"></status>
</kw>
<status status="FAIL" starttime="20190625 20:27:32.274" endtime="20190625 20:27:32.304" critical="yes">No keyword with name 'open browser' found.</status>
</test>
<status status="FAIL" starttime="20190625 20:27:31.778" endtime="20190625 20:27:32.311"></status>
</suite>
<statistics>
<total>
<stat pass="0" fail="1">Critical Tests</stat>
<stat pass="0" fail="1">All Tests</stat>
</total>
<tag>
</tag>
<suite>
<stat pass="0" fail="1" id="s1" name="TC 002 Pharmacy">TC 002 Pharmacy</stat>
</suite>
</statistics>
<errors>
<msg timestamp="20190625 20:27:32.040" level="ERROR">Error in file 'C:\Users\TestingWorld\PycharmProjects\AngularWebAutomation\TestCases\TC_002_Pharmacy.robot': Importing test library 'ExtendedSelenium2Library' failed: ModuleNotFoundError: No module named 'ExtendedSelenium2Library'
Traceback (most recent call last):
  None
PYTHONPATH:
  C:\Python37\Scripts\robot.exe
  c:\python37\python37.zip
  c:\python37\DLLs
  c:\python37\lib
  c:\python37
  c:\python37\lib\site-packages</msg>
<msg timestamp="20190625 20:27:32.125" level="ERROR">Error in file 'C:\Users\TestingWorld\PycharmProjects\AngularWebAutomation\Resources\CommonKewwords.robot': Importing test library 'ExtendedSelenium2Library' failed: ModuleNotFoundError: No module named 'ExtendedSelenium2Library'
Traceback (most recent call last):
  None
PYTHONPATH:
  C:\Python37\Scripts\robot.exe
  c:\python37\python37.zip
  c:\python37\DLLs
  c:\python37\lib
  c:\python37
  c:\python37\lib\site-packages</msg>
<msg timestamp="20190625 20:27:32.252" level="ERROR">Error in file 'C:\Users\TestingWorld\PycharmProjects\AngularWebAutomation\Validations\Verifications.robot': Importing test library 'ExtendedSelenium2Library' failed: ModuleNotFoundError: No module named 'ExtendedSelenium2Library'
Traceback (most recent call last):
  None
PYTHONPATH:
  C:\Python37\Scripts\robot.exe
  c:\python37\python37.zip
  c:\python37\DLLs
  c:\python37\lib
  c:\python37
  c:\python37\lib\site-packages</msg>
</errors>
</robot>
